### ISSUE TITLE
undefined method `cards' for Stripe::Customer

### ISSUE THREAD

**Issue by spacewaffle** [andrewculver/koudoku#101]
Getting this issue when trying to subscribe a user to a plan. This was using the latest version of the Stripe api. Rolling back to an older api version in the Stripe dashboard seems to have fixed things for me. Any idea what's causing this? Maybe a breaking change in the stripe api?

**Comment by yas4891** [comment]
yes. There's a breaking change (https://stripe.com/docs/upgrades) where that attribute is no longer called `card` but `source` instead

**Comment by yas4891** [comment]
Koudoku does not work with API versions newer than `2015-02-16`

**Comment by razvanh** [comment]
@spacewaffle How do you roll back to an older api version? Is that possible?

**Comment by yas4891** [comment]
@razvanh I don't think it is possible to downgrade the API version. 
I wrote Stripe support just for extra clarity.

**Comment by razvanh** [comment]
@yas4891 I don't think it's possible either. Only way would be if you had an older account that you can use. 

Is there any plan to update koudoku to work with the current api version?

**Comment by yas4891** [comment]
@razvanh Feel free to submit a PR!

**Comment by spacewaffle** [comment]
@razvanh login to stripe, head to account settings, api keys, and I get an option to change my api version at the top above where my keys are listed. My stripe account is a bit older though, and judging from what people have said, you may not see it if your account is newer.

**Comment by yas4891** [comment]
@spacewaffle you can only UPGRADE there, no downgrades. 

@razvanh just heard back from Stripe support: You need to email them and they can downgrade you manually

**Comment by razvanh** [comment]
@yas4891 thank you.

**Comment by confact** [comment]
I made a PR for adding support for this API Version, look it up #116

**Comment by hannatime** [comment]
I have had my api version downgraded to 2015-02-16 and am still getting the encoding error.. is there a way to solve this?

**Comment by bval** [comment]
@hannatime I just signed up for a brand new stripe account and it works fine with Koudoku set to the default `Stripe.api_version` of `2015-01-11`. The Stripe API allow the client to request a specific API version. You don't need your account downgraded. See my comment in #134 for an explanation of why you're getting the encoding error and how to resolve it.

Hope this helps,

Brandon
